src/index.js

```diff
@@ -18,6 +18,7 @@ import tr from 'react-intl/locale-data/tr'; // Turkish
 import uk from 'react-intl/locale-data/uk'; // Ukrainian
 import ru from 'react-intl/locale-data/ru'; // Russian
 import it from 'react-intl/locale-data/it'; // Italian
+import mr from 'react-intl/locale-data/mr'; // Italian
 
 import { flattenMessages } from './js/utils'; // flatten messages util function in order to use nested js object for translated texts
 import messages from './messages';
```

Comment:
You also forgot ti import the marathi locale for moment.js

Reply:
Sorry for my ignorance, I'll update the comment